wl/utils/Utils.java

```diff
@@ -481,20 +494,24 @@ public Value(String k, String v, String t) {
 			}
 		}
 
-		TreeMap<String, Value> profile = new TreeMap<>();
+		Profile profile = new Profile();
 		if (!f.isFile()) {
 			_staticprofiles.put(key, profile);
 			return profile;
 		}
 
 		for (String line : Files.readAllLines(f.toPath())) {
 			line = line.trim();
-			if (line.startsWith("#")) continue;
+			if (line.isEmpty() || line.startsWith("#")) continue;
+			if (line.startsWith("[") && line.endsWith("]")) {
+				profile.name = line;
```

Comment:
This should be called "section". 
So this function now supports the `[global]` section, but it will fail, if the profile has multiple sections.
Maybe abort reading the profile, if a section appears and `profile.name != null`. 
Or if you want to go for full support you could use a `TreeMap<String (section), TreeMap<String (key), Value>>` inside the Profile class and add wrapper functions for `put` and `get` which point to the global/default section.